Sources/AppStoreConnectCLI/Services/Operations/ReadAppOperation.swift

```diff
@@ -0,0 +1,25 @@
+// Copyright 2020 Itty Bitty Apps Pty Ltd
+
+import AppStoreConnect_Swift_SDK
+import Combine
+import Foundation
+
+struct ReadAppOperation: APIOperation {
```

Comment:
Do you think this operation should be responsible for getting one app via bundle id or app id?
That way we could stop using the `GetApps` operation to get a single app

🤔